commands_to_load/Administration.py

```diff
@@ -11,18 +11,29 @@ class Administration():
 	def __init__(self, bot):
 		self.bot = bot
 
+	def validCog(self, cog):
+		for name in cogs:

```

Comment:
one last nit pick, changing that to `for cog in cogs` sounds nicer

Reply:
done